lib/puppet/type/ini_subsetting.rb

```diff
@@ -46,16 +45,16 @@ def munge_boolean_md5(value)
     desc 'The separator string between the subsetting name and its value. Defaults to the empty string.'
     defaultto('')
   end
+  # rubocop:enable GetText/DecorateString
 
   newparam(:path) do
     desc 'The ini file Puppet will ensure contains the specified setting.'
     validate do |value|
       unless (Puppet.features.posix? && value =~ %r{^\/}) || (Puppet.features.microsoft_windows? && (value =~ %r{^.:\/} || value =~ %r{^\/\/[^\/]+\/[^\/]+}))
-        raise(Puppet::Error, "File paths must be fully qualified, not '#{value}'")
+        raise(Puppet::Error, _("File paths must be fully qualified, not '%{value}'")) % { value: value }
```

Comment:
As above the line should be:
`raise(Puppet::Error, _("File paths must be fully qualified, not '%{value}'") % { value: value })`